doc/HttpLuaModule.wiki

```diff
@@ -523,12 +523,12 @@ Here is the reason: by design, the global environment has exactly the same lifet
 Generally, use of Lua global variables is a really really bad idea in the context of ngx_lua because
 
 # misuse of Lua globals has very bad side effects for concurrent requests when these variables are actually supposed to be local only,
-# Lua global variables require Lua table look-up in the global environment (which is just a Lua table), which is kinda expensive, and
-# some Lua global variable references are just typos, which are hard to debug.
+# Lua global variables require Lua table look-up in the global environment (which is just a Lua table), which is computationally expensive, and
+# some Lua global variable references may include typing errors which make such difficult to debug.
 
-It's *highly* recommended to always declare them via "local" in the scope that is reasonable.
+It is therefore *highly* recommended to always declare such as "local" within an appropriate scope instead.
```

Comment:
To maintain context here, we could say:
```
It is therefore highly recommended to always declare variables and functions as "local" within an appropriate scope, instead of the default global scope assumed by Lua:

    -- don't
    foo = 123
    -- do
    local foo = 123

    -- don't
    function foo() return 123 end
    -- do
    local function foo() return 123 end
```

This provides a good summary for this paragraph (for non-attentive readers or readers who skipped the details) and provides a concrete, more natural example than the above `require` one. It also shows that this applies to functions (a mistake I have seen too often from contributors).